main.py

```diff
@@ -74,7 +76,8 @@ def search_key():
     done=False #loop variable
 
     howto = False #Instruction Menu
-    start = False #Game Started
+    start = False #Game Start
+    exit = False #Exit

```

Comment:
`exit` is an unnecessary variable because you're anyway calling `pygame.quit()` then `sys.exit()`